bokeh/util/dependencies.py

```diff
@@ -18,7 +23,11 @@ def import_optional(mod_name):
     try:
         return import_module(mod_name)
     except ImportError:
-        return None
+        pass
+    except Exception as exc:
+        msg = "Failed to import optional module `{}`".format(mod_name)
+        logger.exception(msg)
+        
```

Comment:
@dhirschfeld I think this needs to print the exception, since the notion is that that is an expected reason for failure (i.e. not an `ImportError`) also to be consistent with the rest of the codebase it would be `e` instead of `exc`